docker-compose.yml

```
@@ -0,0 +1,47 @@
+    environment:
+      - NODE_ENV=development
+
+  full-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - frontend
+      - backend
+    ports:
+      - "5173:5173" 
```

yamllint

[error] trailing spaces

(trailing-spaces)